spec/enigma_spec.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/enigma'
+
+
+RSpec.describe Enigma do
+  let(:message) { {
+    :message => "hello world", 
+    :key => "02715", 
+    :date => "040895"
+   } }
+  let(:ciphertext) { {
+    :ciphertext => "keder ohulw", 
+    :key => "02715", 
+    :date => "040895"
+  } }
```

Comment:
A better way to write this is to declare it in a `let` block:

```
  let(:ciphertext) do
    {
      :ciphertext => "keder ohulw", 
      :key => "02715", 
      :date => "040895"
    }
  end
```

Reply:
Oh I love that, that do block is way prettier. I am going to remember that one